source/appModules/calculator.py

```diff
@@ -112,6 +128,16 @@ def event_UIA_notification(self, obj, nextHandler, displayString=None, activityI
 				and doNotAnnounceCalculatorResults
 			):
 				return
+			# #16573: when pasting, calculator sends a truncated UIA displayString
+			# (for example "Display is 1" when it should be "Display is 12.34").
+			# To fix this, we ignore the displayString sent via the UIA notification,
+			# and fetch the value directly from the UI element.
+			for child in resultElement.children:
```

Comment:
Good catch - in full screen, expression text is marked as first child, whereas  in compact (mini) mode, actual results text is the first child.

Reply:
Thanks!